### DISCUSSION TITLE
How to change the Window view to the timeline pie chart(like version 0.9.2)?

### DISCUSSION THREAD

**Discussion by sv158** [ActivityWatch/activitywatch#543] [Q&A] [answered]
Recently I update my ActivityWatch to the latest version 0.10.0. However, I found the `Window` view under the `Activity` page is totally changed. 

Personally I think the previous version is more useful, because the view of `Timeline` page is too compact and hard to align with the actual time. Also, the big chart in version 0.9.2 is easy to locate the application's title on each time duration.

Is there anyway to achieve the same effect in version 0.10.0? Currently I've stay in the 0.9.2 version because of this feature.

![image](https://user-images.githubusercontent.com/58453657/103132821-8fdad000-46e1-11eb-89b4-2d1ea4451e67.png)

**Comment by ErikBjare** [comment, marked as answer]
Yeah, sorry about that, it was dropped due to tech debt. Since users seem to miss it, it'll probably be reintroduced eventually. But until then you should stay on v0.9.2 if you really need it.

See this comment for details: https://github.com/ActivityWatch/activitywatch/issues/508#issuecomment-730396764
